### DISCUSSION TITLE
When will 2024.1.0 be published to nuget?

### DISCUSSION THREAD

**Discussion by rhysparry** [sshnet/SSH.NET#1430] [General]
Hi, I noticed the [2024.1.0](https://github.com/sshnet/SSH.NET/releases/tag/2024.1.0) release was made a few days ago and was curious when it would be made available on nuget?

**Comment by Rob-Hague** [comment]
Erm, I had a slight "premature release" which I didn't coordinate very well. The package is built, it should be published in the next day or two

**Comment by WojciechNagorski** [comment]
I've just published the new version.
